src/components/layout/header/Notifications/index.tsx

```diff
@@ -1,14 +1,14 @@
 import { useState } from 'react';
-import clsx from 'clsx';
 import { LegacyNotificationIcon } from '@deriv/quill-icons';
 import { useTranslations } from '@deriv-com/translations';
-import { Notifications as UINotifications, Tooltip, useDevice } from '@deriv-com/ui';
+import { Tooltip } from '@deriv-com/ui';
 import './notifications.scss';
 
 export const Notifications = () => {
     const [isOpen, setIsOpen] = useState(false);
     const { localize } = useTranslations();
-    const { isMobile } = useDevice();
+    // TODO: need to add this back after fixing notifications component
```

Comment:
@rupato-deriv can we create a card for this in backlogs?

Reply:
@sandeep-deriv Bhai I think we may not need to create any card as we were planning to handle it in the account switcher card. This part is having issues with the login for some reason which can be checked in the account switcher card.